lib/stat_tracker.rb

```diff
@@ -0,0 +1,21 @@
+require 'csv'
+class StatTracker
+  attr_reader :games,
+              :teams,
+              :game_teams
+
+  def initialize(games, teams, game_teams)
+    @games      = games#games-hash value
+    @teams      = teams #same as above
+    @game_teams = game_teams #same as above
+  end
+
+
+  def self.from_csv(locations)
+    games = CSV.read(locations[:games], headers: true)
```

Comment:
Nice application from the CSV lesson! Looks good before we add the data to the classes.